.github/workflows/ci-build.yml

```diff
@@ -18,10 +18,10 @@ jobs:
     - name: Lint
       run: cargo fmt --message-format human -- --check
     - name: Build
-      run: cargo build --verbose
+      run: cargo build --verbose --workspace
     - name: Run tests
       run: cargo test --verbose
```

Comment:
`cargo test` no longer runs tests. It appears you also need to pass the `--workspace` flag here.

Reply:
Good catch! I added `"."` to `workspace.default-members` in `Cargo.toml`, and now `cargo test` will run the tests in the root crate (`prio`) without requiring `--all` or `--workspace`.